spec/features/user_signup_spec.rb

```diff
@@ -34,4 +34,29 @@
     end
 
   end
+
+  context "as a borrower" do
+
+    it "can sign up with valid information" do
+      expect(User.count).to eq(0)
+
+      visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.